Motion to approve the Council Webpage CWG (2025-06-05) (#19)
thunderbird/council-docs: WORKING_GROUPS.md

# Council Working Groups
Council working groups (CWGs) are a way to improve focus and effectiveness. Reaching consensus in
a group of seven can be difficult, but smaller groups working on specific topics tend to produce
stronger proposals that are more likely to pass a council vote.

The purpose of a CWG is to explore a topic and bring a clear recommendation back to the full
council. CWGs are not decision-making bodies. Their role is to develop recommendations that
reflect the best interests of the project, consider the broader community, and can gain support
from fellow council members.

Each CWG begins with a charter. This sets the stage by defining what the group will work on and
how. The charter should include enough detail to align expectations, but not so much that it
creates unnecessary debate before the work begins.

## 2025-2026

## Council Webpage Work Group

> <table>
> 
> <tr><td valign="top">Timeframe</td><td>March 15, 2025–July 15, 2025</td></tr>
> <tr><td valign="top">Participants</td><td>Mic, Philipp, Teal</td></tr>
> <tr><td valign="top">Purpose</td><td><p>There has been a perceived lack of external information about what the council is, what it
> does, and why one should get involved. We aim to create a page that gives a high level
> overview:</p>
> <ul>
> <li>What the Council is and what it does</li>
> <li>How one can get involved and join the join the election roll</li>
> <li>How the Council relates to MZLA, the MZLA board, Mozilla, the Mozilla Corporation
>   (MoCo), the Mozilla Foundation (MoFo) and the new nonprofit Mozilla Organization (MoOr?)</li>
> <li>Link to important documents (bylaws, term sheet, election roll, community updates)</li>
> <li>Provide contact information for the Council, as well as information about each member</li>
> </ul></td></tr>
> <tr><td valign="top">Scope</td><td><ul>
> <li>Devise a single page that provides a high level overview of the council</li>
> <li>Work out what information needs to go onto the council page</li>
> <li>Work with MZLA's MarComm Team to ensure the page resonates with our audience</li>
> <li>Have MZLA's Services Team add the page to thunderbird.net</li>
> <li>Find a good place for where to share additional information about the council and set up this page.</li>
> </ul></td></tr>
> <tr><td valign="top">Goals</td><td><ul>
> <li>Information that MZLA can turn into a council web page</li>
> <li>Present a prototype for the page with additional information to the council and agree on necessary changes.</li>
> </ul></td></tr>
> <tr><td valign="top">Risks</td><td><ul>
> <li>If we pack too much information into the web page, we risk losing focus and not
>   appealing to the community audience</li>
> </ul></td></tr>
> </table>

Reviewer: Maybe a Markdown table would be more concise/readable:
```suggestion
| Field | Details |
| --- | --- |
| **Timeframe** | March 15, 2025 – July 15, 2025 |
| **Participants** | Mic, Philipp, Teal |
| **Purpose** | <ul><li>What the Council is and what it does</li><li>How one can get involved and join the election roll</li><li>How the Council relates to MZLA, the MZLA board, Mozilla, the Mozilla Corporation (MoCo), the Mozilla Foundation (MoFo) and the new nonprofit Mozilla Organization (MoOr?)</li><li>Link to important documents (bylaws, term sheet, election roll, community updates)</li><li>Provide contact information for the Council, as well as information about each member</li></ul> |
| **Scope** | <ul><li>Devise a single page that provides a high-level overview of the Council</li><li>Work out what information needs to go onto the Council page</li><li>Work with MZLA’s MarComm Team to ensure the page resonates with our audience</li><li>Have MZLA’s Services Team add the page to thunderbird.net</li><li>Find a good place to share additional information about the Council and set up this page</li></ul> |
| **Goals** | <ul><li>Information that MZLA can turn into a Council web page</li><li>Present a prototype for the page with additional information to the Council and agree on necessary changes</li></ul> |
| **Risks** | <ul><li>If we pack too much information into the web page, we risk losing focus and not appealing to the community audience</li></ul> |
```

Author: Markdown table means we need to have an unnecessary header. The HTML here is generated anyway, and what counts more to me is how it is visualized.
